docker/build/devdeps.Dockerfile

```diff
@@ -87,6 +88,11 @@ SHELL ["/bin/bash", "-c"]
 ARG DEBIAN_FRONTEND=noninteractive
 ENV HOME=/home SHELL=/bin/bash LANG=C.UTF-8 LC_ALL=C.UTF-8
 
+# Copy over doxygen.
+COPY --from=doxygenbuild /usr/local/bin/doxygen /usr/local/bin/doxygen
```

Comment:
Mentioned this earlier but it would be good to move this before the LLVM build since this won't really ever change. That way when the cache changes we shouldn't have to rebuild this

Reply:
I assume you mean the build stage rather than the copy here. I move it up, though I would have expected that the location of the COPY statement is relevant (which is already at the top) rather than the location of the build stage (which has no dependency on any earlier stage). However, I do see doxygen getting rebuilt when I wouldn't expect it to, so at some point I'll look in more detail at why that is...